examples/with-docker/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:16-alpine AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
-COPY package.json yarn.lock ./
+COPY package.json yarn.lock* package-lock.json* ./
 RUN yarn install --frozen-lockfile
```

Comment:
Why is the `package-lock.json` copied and then `yarn` is used on the next line? Maybe this can be a comment instead?

```suggestion
COPY package.json yarn.lock /
RUN yarn install --frozen-lockfile

# If using nom with a `package-lock.json` comment out above and use below instead
# COPY package.json package-lock.json / 
# RUN npm install
```

Reply:
Thank you, I hadn't noticed that at all😅